src/compressed_tensors/quantization/quant_scheme.py

```diff
@@ -48,6 +49,7 @@ class QuantizationScheme(BaseModel):
     weights: Optional[QuantizationArgs] = None
     input_activations: Optional[QuantizationArgs] = None
     output_activations: Optional[QuantizationArgs] = None
+    format: Optional[str] = None
```

Comment:
Missing docstring

Reply:
To be clear, we should support users overriding format, right? This is a useful if, for example, we want to run a model which has been fake quantized for debugging purposes, and then also to support different packing formats